main/grep/APKBUILD

```diff
@@ -23,7 +22,6 @@ prepare() {
 }
 
 build() {
-	cd "$builddir"
```

Comment:
Why are you removing the `cd` command here?

Reply:
Because I thought it was acceptable from after the release of AL3.9. If that is not the case I will revert it.